src/combination.jl

```diff
@@ -1,14 +1,35 @@
 """ This file contains all the functions related to Bliss combinations. """
 
 ######---------------- Functions for Bliss combination ----------------########
-function CombinationParam(p1::Array{Float64, 2}, p2::Array{Float64, 2}, n::Int)
+
+""" Unit function to calculate the bliss for 2 drugs at one specific concentration. """
+function Bliss_params_unit(pp1, pp2, control)
+    p1 = copy(pp1)
+    p2 = copy(pp2)
+    # normalization
+    p1[1:2] .= 1.0 .- (pp1[1:2] ./ control[1:2, 1]) # g1 and g2 prog. rates
+    p1[3:4] .= pp1[3:4]                          # g1 and g2 death rates
+    # drug B
+    p2[1:2] .= 1.0 .- (pp2[1:2] ./ control[1:2, 2])
+    p2[3:4] .= pp2[3:4]
+
+    c = Array{eltype(pp1), 1}(undef, 9)
+    c[1:2] .= (1.0 .- (p1[1:2] .+ p2[1:2] .- p1[1:2] .* p2[1:2])) .* ((control[1:2, 1] .+ control[1:2, 2]) ./ 2)
+    c[3:4] .= p1[3:4] .+ p2[3:4]
+ 
+    c[5:9] .= pp1[5:9]
+    return c
+end
+
```

Comment:
I will use this and edit the function that calculates the Bliss combination for parameters. I needed this because I wanted to only pass the EC50 parameters to it.